py/server/deephaven/table.py

```diff
@@ -251,6 +251,66 @@ def with_filters(self, filters: Union[str, Filter, Sequence[str], Sequence[Filte
         except Exception as e:
             raise DHError(e, "with_filters operation on RollupTable failed.") from e
 
+    def layout_hints(self, front: Union[str, List[str]] = None, back: Union[str, List[str]] = None,
+                     freeze: Union[str, List[str]] = None, hide: Union[str, List[str]] = None,
+                     column_groups: List[dict] = None, search_display_mode: SearchDisplayMode = None) -> Table:
```

Comment:
`dict` isn't a great type hint because it doesn't indicate the contents of the dict.  It should include the acceptable key and value types.